pom.xml

```diff
@@ -281,25 +281,6 @@ THE SOFTWARE.
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-enforcer-plugin</artifactId>
-        <configuration>
-          <rules>
-            <requireUpperBoundDeps>
-              <excludes combine.children="append">
-                <!--
-                  Stapler requests Guava 14.0 and Jenkins core requests Guice 4.0 which requests
-                  Guava 16.0.1. Core actually provides 11.0.1. Work around this mess by just
-                  excluding Guava from the RequireUpperBoundDeps check. The long-term fix is
-                  tracked in JENKINS-36779.
```

Comment:
https://issues.jenkins.io/browse/JENKINS-36779 closed in 2.320. For now https://github.com/jenkinsci/jenkins-test-harness/blob/e5ab6d25ed227d33a98b39f2fb5aef9a5f233fc8/pom.xml#L59 but we will probably need to move to 2.361.x at some point (#453).